server/routes/api/observation_dates.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from flask import Blueprint, request
+import services.datacommons as dc
+
+# Define blueprint
+bp = Blueprint("observation-dates", __name__)
+
+
+@bp.route('/api/observation-dates')
+def observation_dates():
+    """
+    Given ancestor place, child place type and stat vars, return the dates that
+	have data for each stat var across all child places.
+    """
+    ancestor_place = request.args.get('ancestorPlace')
```

Comment:
since within place endpoints all use the params `parentEntity`, `childType`, and `variables`, should we keep params consistent and use those same parameters?